docker-compose.yml

```diff
@@ -50,6 +50,7 @@ services:
     command: ["/app/run-celery.sh"]
     environment:
       << : *shared-app-environment
+      CELERY_QUEUE_NAMES: written_store,inbound_store,send
```

Comment:
Please also add this to the Helm chart which is used for the deployment to Kubernetes so that the correct configuration values are also set in production and then this is ready to merge. Thanks!

The environment variables for the worker on k8s are configured in [worker-deployment.yaml](https://github.com/ascoderu/opwen-cloudserver/blob/9825978431ff19c23b6761755fa373d836adcd02/helm/templates/worker-deployment.yaml#L24), like this:

```yaml
env:
  - name: CELERY_QUEUE_NAMES
    value: written_store,inbound_store,send
```

Reply:
will do!